src/condor_vm-gahp/xen_type.linux.cpp

```diff
@@ -1273,6 +1282,16 @@ XenType::checkXenParams(VMGahpConfig* config)
 				fixedvalue.c_str());
 		return false;
 	}
+
+	virConnectPtr libvirt_connection = virConnectOpen("xen:///");
```

Comment:
https://libvirt.org/uri.html seems to indicate that this should be `xen:///system`?

Reply:
I concur with your assessment. `xen:///` is what the vm-gahp uses elsewhere in the code. I tested kvm, and xen doesn't appear to work on the CHTC machines. I want to say leave it alone unless someone complains, which hopefully own't happen before we decide to rip all of this code out.

Reply:
Better to be consistent with the rest of the code; if it's broken, that way at least it'll be broken in the same way (that is, the test will be for what we're actually doing) and that will make it easier to fix everywhere if we need to.